src/pkg/post/get_post.go

```diff
@@ -0,0 +1,37 @@
+package post
+
+import (
+	"net/http"
+
+	"github.com/Johannes-Krabbe/hive-nexus-api/src/models"
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+)
+
+func (h handler) GetPost(c *gin.Context) {
+	// getting params
+	postID := c.Query("postId")
+
+	if postID == "" {
+		c.AbortWithStatusJSON(http.StatusBadRequest, "Include postId in query params")
+		return
+	}
+
+	var post models.Post
+	if postID != "" {
```

Comment:
This check here if postID != "" is unnecessary since it will always be true when this line is executed.

Reply:
Fixed :)